integration/emr/alluxio-emr.sh

```diff
@@ -668,9 +668,10 @@ IN
   local -r emr_version=$(jq ".releaseLabel" /mnt/var/lib/info/extraInstanceData.json  | sed -e 's/^"emr-//' -e 's/"$//')
```

Comment:
update the comments before this line? and explain why the behavior for 5.30+ is different